lib/omnibus/packagers/bff.rb

```diff
@@ -61,7 +61,7 @@ def package_name
     # @return [String]
     #
     def scripts_install_dir
-      File.expand_path("#{project.install_dir}/embedded/share/installp")
+      File.expand_path('embedded/share/installp', project.install_dir)
```

Comment:
We can use `File.join`, but expanding via relative paths was basically ripped out of omnibus because of the unexpected behavior with relative vs absolute paths vs the cwd.